internal/cmd/beta/server/create/create.go

```diff
@@ -0,0 +1,338 @@
+				cobra.CheckErr(err)
+			}
+
+			if bootVolumeSourceId == "" && bootVolumeSourceType == "" {
+				err := cmd.MarkFlagRequired(imageIdFlag)
+				cobra.CheckErr(err)
+			}
+
+			if imageId != "" && (bootVolumeSourceId != "" || bootVolumeSourceType != "") {
+				p.Warn("Image ID flag cannot be used together with any of the boot volume flags.\n")
+			}
+		},
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			ctx := context.Background()
+			model, err := parseInput(p, cmd)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			projectLabel, err := projectname.GetProjectName(ctx, p, cmd)
+			if err != nil {
+				p.Debug(print.ErrorLevel, "get project name: %v", err)
+				projectLabel = model.ProjectId
+			}
+
+			if !model.AssumeYes {
+				prompt := fmt.Sprintf("Are you sure you want to create a server for project %q?", projectLabel)
+				err = p.PromptForConfirmation(prompt)
+				if err != nil {
+					return err
+				}
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			resp, err := req.Execute()
+			if err != nil {
+				return fmt.Errorf("create server : %w", err)
+			}
+			serverId := *resp.Id
+
+			// Wait for async operation, if async mode not enabled
+			if !model.Async {
+				s := spinner.New(p)
+				s.Start("Creating server")
+				_, err = wait.CreateServerWaitHandler(ctx, apiClient, model.ProjectId, serverId).WaitWithContext(ctx)
+				if err != nil {
+					return fmt.Errorf("wait for server creation: %w", err)
+				}
+				s.Stop()
+			}
+
+			return outputResult(p, model, projectLabel, resp)
+		},
+	}
+	configureFlags(cmd)
+	return cmd
+}
+
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().StringP(nameFlag, "n", "", "Server name")
+	cmd.Flags().String(machineTypeFlag, "", "Name of the type of the machine for the server. Possible values are documented in [Virtual machine flavors](https://docs.stackit.cloud/stackit/en/virtual-machine-flavors-75137231.html)")
+	cmd.Flags().String(affinityGroupFlag, "", "The affinity group the server is assigned to")
+	cmd.Flags().String(availabilityZoneFlag, "", "The availability zone of the server")
+	cmd.Flags().String(bootVolumeSourceIdFlag, "", "ID of the source object of boot volume. It can be either 'image-id' or 'volume-id'")
+	cmd.Flags().String(bootVolumeSourceTypeFlag, "", "Type of the source object of boot volume. It can be either  'image' or 'volume'")
+	cmd.Flags().Int64(bootVolumeSizeFlag, 0, "The size of the boot volume in GB. Must be provided when `source_type` is `image`")
+	cmd.Flags().String(bootVolumePerformanceClassFlag, "", "Boot volume performance class")
+	cmd.Flags().Bool(bootVolumeDeleteOnTerminationFlag, false, "Delete the volume during the termination of the server. Defaults to false")
+	cmd.Flags().String(imageIdFlag, "", "The image ID to be used for an ephemeral disk on the server. Either image-id or boot volume is required")
```

Comment:
```suggestion
	cmd.Flags().String(bootVolumeSourceIdFlag, "", "ID of the source object of boot volume. It can be either an image or volume ID")
	cmd.Flags().String(bootVolumeSourceTypeFlag, "", "Type of the source object of boot volume. It can be either  'image' or 'volume'")
	cmd.Flags().Int64(bootVolumeSizeFlag, 0, "The size of the boot volume in GB. Must be provided when 'boot-volume-source-type' is 'image'")
	cmd.Flags().String(bootVolumePerformanceClassFlag, "", "Boot volume performance class")
	cmd.Flags().Bool(bootVolumeDeleteOnTerminationFlag, false, "Delete the volume during the termination of the server. Defaults to false")
	cmd.Flags().String(imageIdFlag, "", "The image ID to be used for an ephemeral disk on the server. Either 'image-id' or 'boot-volume-...' flags are required")
```